app/components/DeleteTodo.js

```diff
@@ -10,6 +10,12 @@ class DeleteTodo extends Component {
         this.state = {}
     }
 
+    function update() {
+      const x = 1;
+      this.update();
+      x++;
+    }
+  
```

Comment:
In the 'deletetodo' method, 'delete_todos' is called with an object containing 'variables' and 'refetchQueries'. Check if 'this.props' is the correct data to pass as 'variables'.